lib/index.js

```diff
@@ -0,0 +1,9 @@
+const binding = require("pkg-prebuilds")(
+    require("path").resolve(__dirname, ".."),
+    require("./options.js")
+);
+
+Object.keys(binding).forEach(function (key)
```

Comment:
wouldn't this just work?
```js
module.exports = binding;
```

Reply:
Thought so as well, the documentation for pkg-rebuilds did it like this as well so I just went with it, will update it later